README.md

```diff
@@ -56,20 +65,44 @@ schema "users" do
 end
 ```
 
-Each field is defined as an [Ecto type](https://hexdocs.pm/ecto/Ecto.Type.html), with the relevant callbacks. So when you call `Ecto.Changeset.cast/4` in your schema's changeset function, the field will be correctly validated. For example, calling cast on the `:email` field will ensure it is a valid format for an email address.
-
-When you load one of the fields into your database, the corresponding `dump/1` callback will be called, ensuring it is inserted into the database in the correct format. In the case of `Fields.EmailEncrypted`, it will encrypt the email address using a give encryption key (set in your config file) before inserting it.
-
-Likewise, when you load a field from the database, the `load/1` callback will be called, giving you the data in the format you need. `Fields.EmailEncrypted` will be decrypted back to plaintext.
-
-Each Field optionally defines an `input_type/0` function. This will return an atom representing the `Phoenix.HTML.Form` input type to use for the Field. For example, `Fields.DescriptionPlaintextUnlimited.input_type` returns `:textarea`.
-
-The fields `DescriptionPlaintextUnlimited` and `HtmlBody` use html_sanitize_ex
-(https://github.com/rrrene/html_sanitize_ex) to remove scripts and help keep your
-project safe. `HtmlBody` is able to display basic html elements whilst
-`DescriptionPlaintextUnlimited` displays text. Remember to use `raw` when rendering
-the content of your `DescriptionPlaintextUnlimited` and `HtmlBody` fields so that
-symbols such as & (ampersand) and Html are rendered correctly. E.g.
+Each field is defined as an 
```

Comment:
I may have got slightly carried away with tidying up the `README.md` ... 🙄 
I didn't change much of the _content_ ...
just added line breaks for readability.